e2e/wallets.po.ts

```diff
@@ -24,119 +24,62 @@ export class WalletsPage {
   }
 
   createWalletCheckValidationSeed() {
-    const label = element(by.css('[formcontrolname="label"]'));
-    const seed = element(by.css('[formcontrolname="seed"]'));
-    const confirm = element(by.css('[formcontrolname="confirm_seed"]'));
     const btnCreate = element(by.buttonText('Create'));
-
-    return label.clear().then(() => {
-      return label.sendKeys('Test wallet').then(() => {
-        return seed.clear().then(() => {
-          return seed.sendKeys('test test2').then(() => {
-            return confirm.clear().then(() => {
-              return confirm.sendKeys('skycoin-web-e2e-test-seed').then(() => {
-                return btnCreate.isEnabled();
-              });
-            });
-          });
-        });
-      });
-    });
+    this.fillWalletForm('Test wallet', 'test test2', 'skycoin-web-e2e-test-seed');
+    return btnCreate.isEnabled();
   }
 
   loadWalletCheckValidationSeed() {
```

Comment:
Now this function does not check the condition correctly. Before the changes it put a label and left the seed empty, to check if the seed was being validated correctly. Now it leaves both the label and the seed empty, so the validation of the seed could be failing, but the test would pass correctly because the button would be disabled because the label is empty.

Reply:
Fixed, rolled it back.